### DISCUSSION TITLE
Version 1.2.99.0-adoption - `<Thank you Matheo G>` edition

### DISCUSSION THREAD

**Discussion by zer0Kerbal** [zer0Kerbal/InSight#37] [Announcements]
[![InSight Lander (INSIGHT)][MOD:shd:stat]][MOD:forum]  
[![KSP version][KSP:shd:stat]][KSP:url] [![License][LIC:shd]][LIC:url]  
![AVC .version files][AVCVAL:shd] [![Pages][MOD:pages:shd]][MOD:pages]

# Version 0.2.99.0adoption - `<Thank you Matheo G>` edition

* Released
  * on 04 Dec 2022
  * for Kerbal Space Program 1.12.4
  * by [zer0Kerbal](https://github.com/zer0Kerbal)

<div style="border:0.5px solid Tomato; background-color: #bada55; color: #FF0000; text-align:center"><h4>
<b>For best results - DO A CLEAN INSTALL; DELETE EXISTING INSTALLATION THEN RE-INSTALL</b></h4><p>Download from <a href="https://www.curseforge.com/kerbal/ksp-mods/InSight/files">CurseForge</a></p></div>

### Summary

* engines
  * stats need review (all stats)
  * there are twelve (12) engines on this part, each with its own EngineFX and effect modules -- only one seems to show.
* Real (Sized) Solar System patch needs much refinement
* models (.mu) have textures baked in
* convert to stock leg modules? (beyond me)
* should include?
  * stock surface scanner module?
  * basic science ?
  * a custom experiement (would need to be provided)

### Archival Releases

* 0.2.0.0-release - `<Archival>` edition

### Create Legal Mumbo Jumbo

* Create Legal Mumbo Jumbo
  * license check
  * offline documentation
  * _Legal
    * screenshots and pdfs
    * adoptionLetters
    * communications concerning
  * _Links/
    * link(s) saved
  * docs/LegalMumboJumbo
    * [License.md]
    * FORUM-##.png's
      * public documentation
  * GitHub: :octocat:
    * LICENSE
    * [license].txt
  * CurseForge
  * SpaceDock
  * CKAN

### Create

* Documentation
  * readme
  * deploy to:
    * CurseForge Description page 🤬
    * Forum Original Post 🐰
    * SpaceDock Information page 🌮
  * release notes
  * [changelog.md]
  * update /docs/
* HeroLogo.png
  * copy/convert to HeroLogo.jpg
* Agency - uses Klockheed Martian
* closes #6 - Create HeroLogo.png
* GitHub Pages
  * docs/
    * [`_config.yml`]
    * [Attribution.md] v1.0.7.1
    * [ManualInstallation.md] v1.1.8.0
    * [404.md] v1.0.3.2
    * [LegalMumboJumbo.md] v1.0.5.1
    * [Localizations.md] v1.1.7.0
    * [Marketing.md] v1.0.1.0
    * [Notices.md] v1.0.1.0
    * [PartsCatalog.md] v1.1.4.1
    * [Why.md] v1.1.0.0
* closes #5 - Create GitHub Pages

### Update Parts

* linting
* modernizing
* and so much more
* add part modules
  * transmitter
  * sas

### Asset Updates

* create Assets/ folder
* Asset conversions
  * convert from mesh to MODEL {}
  * No textures (baked into model)
* rename
  * models to unique names
* update
  * model pointers (.png et al to .dds)
* relocate assets to Assets/
* nothing to eliminate (duplicates)
* relocate part.cfg to Parts/
* closes #9 - Part Asset Updates

### Localization

* Create
  * directory and contents
  * Localization/
    * <en-us.cfg>
    * [readme.md] v2.1.2.0
    * [quickstart.md] v1.0.1.1
* Add localized tags to parts
* Create
  * <GhostParts.cfg> v1.3.0.0
    * part names have changed
* closes #8 - Create Localization directory and contents
* closes #14 - English <en-us.cfg>
* closes #31 - Part Localization
* closes #32 - Part Tags
* updates #13 - Localization - Master

### Compatibility

* Create
  * [RealSolarSystem.cfg] v1.0.0.0
  * [kOS.cfg] v1.0.0.0

### Update License

* Updated License:
  * to GPL-2.0
  * was: WTFPL
* closes #10 - Update License

### Status

* Issues
  * closes #1 - InSight Lander (INSIGHT) 0.2.99.0-adoption `<Thank you Matheo G>` edition
  * closes #2 - 1.1.99.0 Create Legal Mumbo Jumbo
  * closes #3 - 1.1.99.0 Create Documentation
  * closes #4 - 1.1.99.0 Create Social Media Presence
  * closes #11 - Create <InSight.cfg>
  * closes #33 - Create Agency

---

## See More

* [Pages][MOD:pages] for so much more (WIP)
* [ChangeLog][MOD:chlog] for more details of changes
* [Discussions][MOD:discu] or [KSP Forums][MOD:forum] for discussions and news
* [Known Issues][MOD:issue] for known issues and feature requests

## Localization

>* ![English][EN] English
>* ***your translation here***
>
> HELP WANTED - See the [README in the Localization folder][lreadme] or the [Quickstart Guide][qstart] for instructions for adding or improving translations. [GitHub][GitHub:url] push is the best way to contribute. *Additions and corrections welcome!*

<div style="border:0.5px solid Tomato; background-color: #BADA55; color: #FF0000; text-align:center">
  <p><b>red box below is a link to forum post on how to get support</b></p>
  <a href="https://forum.kerbalspaceprogram.com/index.php?/topic/83212-*">
    <p><img src="https://i.postimg.cc/vHP6zmrw/image.png" alt="How to get support"></p></a>
  <p style="color: #000000;">Be Kind: Lithobrake, not jakebrake! Keep your Module Manager up to date</p>
</div>

---

### How to support this and other great mods by [`zer0Kerbal`][zer0Kerbal]  

[![Support][PAYPAL:img]][PAYPAL:url] [![Patreon][PATREON:img]][PATREON:url] [![Github Sponsor][GSPONS:img]][GSPONS:url] [![Buy zer0Kerbal a snack][BMCC:img]][BMCC:url]  

#### Release Schedule

1. GitHub, reaching first manual installers and users of KSP-AVC. Right now.  
2. CurseForge. Right now.  
3. SpaceDock (and CKAN users). Soon™  

[MOD:chlog]: https://raw.githubusercontent.com/zer0Kerbal/InSight/master/changelog.md "Changelog"
[MOD:discu]: https://github.com/zer0Kerbal/InSight/discussions/ "Discussions"
[MOD:forum]: https://forum.kerbalspaceprogram.com/index.php?/topic/210824-*/ "InSight Lander (INSIGHT)"
[MOD:issue]: https://github.com/zer0Kerbal/InSight/issues/ "Issue Tracker"
[MOD:pages]: https://zer0kerbal.github.io/InSight/ "GitHub Pages"
[MOD:pages:shd]: https://img.shields.io/badge/GitHub-Pages-white?style=plastic&labelColor=9cf&logoColor=181717&logo=github/ "GitHub IO"

[MOD:shd:stat]: https://img.shields.io/badge/InSight%20Lander%20(INSIGHT)%20-v0.2.99.0--adoption-BADA55.svg?style=plastic&labelColor=darkgreen/ "0.2.99.0adoption"

[GITHUB:url]: https://github.com/zer0Kerbal/InSight/ "GitHub"

[KSP:url]: http://kerbalspaceprogram.com/ "Kerbal Space Program"
[KSP:shd:stat]: https://img.shields.io/badge/KSP-1.12.4-blue.svg?style=plastic&logo=data:image/webp;base64,UklGRpAGAABXRUJQVlA4TIQGAAAvH8AHEE0obNsGDakwXkT/Q4chj76jn1yYjSRjH6H+6xQqRUkkSc5ckHD+NUUVhx4+RNu2bTTl/3OTVhD6H8u6fF8dDYm40CK7N0CjAxK52rYtjfQRAvGNjbu7V+vuLp2fwhzA6gG4li6de+nuO+4uJBlkwoTA//90kENgtZZs27ZpO2Oufa5vbNsq2Sw6qaa9r3n/kZJTex/wSrFt27o62kuSJNuqrdTa1+/7uLu7uzQZAk0mRN/mAz2CAbi723vvy5WztxxIAAiw+dh2bdvWZJuT12y1bdu2bdu2bbs3AQABlsED+y50lDqLE4pf/Uxe8KO77HFes9hvxu1p3O2Q09LVg0NVjo7Z5U6AvwBSOfivKdQTujOnXNG8hzNhqYSMMHJ+MwWWyTSjNUoP1jWHp1ZNmgB8TMH3Tl33mvoVb8uIj3umyIJLsAmYoCIqUCVMCN5WrW78Qi+AkeaGOkWFk3QbRmIWHpIZSURXDxhJVlJiYZ6kLEgeEjIRCBmxebhCTWlTJNaXLystGysVToPOAFJJjqSPNOXftSVbXPQjxrzVaFNzXvBonWJlonIVTJabAfjZ0Jxufl5GPntEHKjB7PIXtg6eYHywJn3hTdijjXUain94KjfCfg1hM3AZRR4N16VMXBrsDVYDCjOIMmGFsAab8rdY7+kIw0BgtRgL1DKJeuNZdQ9f5RAGGfOQfZxexj7t37RkjNp4rPDIHOJVyPBF+XHKsnQk4SWRBl8Wot74WtkBqidQggbyj2vW1WEFG6JfUZ8UxhVzOmAUdKTYZ3yxR2QwIIAQdA9YgiRBZWyCguiOER9y9IauiyGqR0EayVLiEVwmXmF5+Z2jfohj8i1q8ybRahCBAXZBCBUqLgAACQFzp5M/o5wzCDcCJZEOF9EzggM7NEd18c1Q85gPBldtg6mwB1N+hzyTSFp5jWM+WpCRBEBAjIaUxJI80o3s6vDYJ7gNXuAUfOacvsLvLhNIMaMwh8GccsXjtuiK+wkxZ/kVdOCzoCeAOn+N7H1rU1YT35MgAWY9yD5wMWI7j7G976bSh8LxDx2jI3IJRAnJ+RQQIU7gABnLd3vKSyMAd71mq7HgV3AJ+Sxh5Bz5O0z/gbTiNT8DxQssrrv7Uyx9Y7q6BQVUoIGonF/FfoC/RqwVHlviCRSABDvLaCUfZlz1cNENn6adNTz+AF8v8KWQAFL+QMkpCthFkpvvq7+2/to6AIBhyvPdrUr3g1XwBEJPMeP/98SxWWvs3nHIhgNnsY1RWVyw+Guh5IZLgj9WzPi/iEk9gPPGLAni3aRlsqlGQ9jP4z9Q/s/xi3dw486abm6Bqi2l3qu9e9tQsdVMdG4zbVYqp5wJcBSA2W6sNmkwxuNk+nw3r4MFi/4EkTAhiRFk70jkqNDiWbDK65OLIn0s0wD/ucAp3XwVSsSNBAqjFUIXwFE0MihZQ6utQYV2oR+tShO7Ad6bswY3xd7qe0VrVxK9ZjueQy4TPnF8MCQGjoxSugSFgFMV4LiVwgsn/i+gXZ2FzplAduxnn0/OlW0uqf+M64MRPqzwrl+PnpKQSaXS49Ui7n2/ctFtHtAtiprzxq6WniwESvW5yUG1Xx6/8Hx8NKELwKZK15pV/EvXvm9ZMlx0aKUO98iUPaWvst/n8ZbOmkvGhcOQRWf5zj9dk9cfpad5oHN3Rns/wsuvy2puxz1Ziu96Q7/SOWoROzvNzFo5Z5+1BDej3OjQ/XymEkW9jr0em5g5SdX8VC2gf9xJb/RWCC5bIKWDgWcYf+K9Kje3zbQBh/F448wMLoICeUyJ330nXlPmawiRT/sblG4vWrbErgQaMzYbZcwbhSaNrwH+Tqa04jqrd3JZTvwbFxHFSVMAv5UZdEq+tQUupcis/5+MZNsxk9b8TPa7cMqdzzrh9FtD5v+vPACvJy7nDT69IP/Yx6EywGdTFsD5iU7bqkovJogzTjQm3iFTyp4jV4bjVKdcnv5/JrhokmpnGAIA4D/AXYCfVgoBXnrDkCqqCHRG529HeYB51Jy1z6nlW/gnVmzyxmVHxnQrxXxelcI0yN85udPl+//t2rzKzA+oluPTNjp6qY1PVduFVdo8ya+8E6p8KOZR+bLj6Vju9oi5dar0erTS8Z1x3/IITU3vyDRLiZWBZVH6CbqURTeLptD3pEPIR4W4QlHfTnRJzZBRJ8MlI8LmmEXLAdAxsqIYbSDGTt65GfF0cUL6aQQ=&labelColor=black/ "Kerbal Space Program"

[LIC:url]: https://www.gnu.org/licenses/gpl-2.0-standalone.html "GPL-2.0"
[LIC:shd]: https://img.shields.io/badge/License-GPL--2.0-A42E2B?labelColor=white&style=plastic&logoColor=A42E2B&logo=gnu "GPL-2.0"

[AVCVAL:shd]: https://github.com/zer0Kerbal/InSight/actions/workflows/AVC-VersionFileValidator.yml/badge.svg?style=plastic&labelColor=black "AVC Valid"

[PAYPAL:img]: https://img.shields.io/badge/Buy%20me%20some%20-LFO-BADA55?style=for-the-badge&logo=paypal&labelColor=FFDD00 "PayPal"
[PAYPAL:url]: https://www.paypal.com/donate?hosted_button_id=DC22YHMEJREKL "PayPal"
[PATREON:img]: https://img.shields.io/badge/Patreon%20-Patreonize-FF424D?style=for-the-badge&logo=patreon "Patreon"
[PATREON:url]: https://www.patreon.com/bePatron?u=23390503 "Patreon"
[GSPONS:img]: https://img.shields.io/badge/Github%20-Sponsor-EA4AAA?style=for-the-badge&logo=githubsponsors "Github Sponsors"
[GSPONS:url]: https://github.com/sponsors/zer0Kerbal "Github Sponsors"
[BMCC:img]: https://img.shields.io/badge/Buy%20Me%20a%20-Snack!-FFDD00?style=for-the-badge&logo=buymeacoffee "Buy Me A Snack"
[BMCC:url]: https://buymeacoffee.com/zer0Kerbal "Buy Me A Snack"
[lreadme]: https://github.com/zer0Kerbal/zer0Kerbal/blob/master/Localization/readme.md "Localization Readme"
[qstart]: https://github.com/zer0Kerbal/zer0Kerbal/blob/master/Localization/quickstart.md "Quickstart"
[EN]: https://raw.githubusercontent.com/zer0Kerbal/zer0Kerbal/master/img/EN.png "English"
[BR]: https://raw.githubusercontent.com/zer0Kerbal/zer0Kerbal/master/img/BR.png "Português Brasil"
[CN]: https://raw.githubusercontent.com/zer0Kerbal/zer0Kerbal/master/img/CH.png "中文"
[DE]: https://raw.githubusercontent.com/zer0Kerbal/zer0Kerbal/master/img/DE.png "Deutsch"
[ES]: https://raw.githubusercontent.com/zer0Kerbal/zer0Kerbal/master/img/ES.png "Español"
[FR]: https://raw.githubusercontent.com/zer0Kerbal/zer0Kerbal/master/img/FR.png "Français"
[IT]: https://raw.githubusercontent.com/zer0Kerbal/zer0Kerbal/master/img/IT.png "Italiano"
[JA]: https://raw.githubusercontent.com/zer0Kerbal/zer0Kerbal/master/img/JA.png "日本語"
[KO]: https://raw.githubusercontent.com/zer0Kerbal/zer0Kerbal/master/img/KO.png "한국어"
[MX]: https://raw.githubusercontent.com/zer0Kerbal/zer0Kerbal/master/img/MX.png "Mexicano Español"
[NL]: https://raw.githubusercontent.com/zer0Kerbal/zer0Kerbal/master/img/NL.png "Dutch"
[NO]: https://raw.githubusercontent.com/zer0Kerbal/zer0Kerbal/master/img/NO.png "Norsk"
[PO]: https://raw.githubusercontent.com/zer0Kerbal/zer0Kerbal/master/img/PO.png "Polski"
[RU]: https://raw.githubusercontent.com/zer0Kerbal/zer0Kerbal/master/img/RU.png "Русский"
[SW]: https://raw.githubusercontent.com/zer0Kerbal/zer0Kerbal/master/img/SW.png "Svenska"
[TR]: https://raw.githubusercontent.com/zer0Kerbal/zer0Kerbal/master/img/TR.png "Türk"
[TW]: https://raw.githubusercontent.com/zer0Kerbal/zer0Kerbal/master/img/TW.png "国语"

[zer0Kerbal]: https://forum.kerbalspaceprogram.com/index.php?/profile/190933-*/ "zer0Kerbal"

## What's Changed
* Version 1.2.99.0-adoption - `<Thank you Matheo G>` edition by @zer0Kerbal in https://github.com/zer0Kerbal/InSight/pull/36

**Full Changelog**: https://github.com/zer0Kerbal/InSight/compare/0.2.0.0-release...1.2.99.0-adoption

<hr /><em>This discussion was created from the release <a href='https://github.com/zer0Kerbal/InSight/releases/tag/1.2.99.0-adoption'>Version 1.2.99.0-adoption - `<Thank you Matheo G>` edition</a>.</em>

**Comment by Carlene12** [comment]
This discussion is very helpful thanks for sharing. You can get help with. [calculador de paypal mexico 2023](https://calculadorapp.com/)
